.github/workflows/update-docker-requirements.yml

```
@@ -42,10 +43,22 @@ jobs:
           NEEDS_UPDATE=$(git diff --quiet && echo 'false' || echo 'true')
           echo "needs-update=${NEEDS_UPDATE}" | tee --append $GITHUB_OUTPUT
 
+      - name: Get last merged PR info
+        id: pr_info
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+        run: |
+          LAST_MERGED_PR=$(gh pr list --repo $GITHUB_REPOSITORY --state closed --json number --jq '.[0].number')
+          echo "Last merged PR number: $LAST_MERGED_PR"
+          PR_INFO=$(gh pr view $LAST_MERGED_PR --repo $GITHUB_REPOSITORY --json author,mergedBy --jq '{author: .author.login, mergedBy: .mergedBy.login}')
+          echo "PR Info: $PR_INFO"
+          echo "pr_author=$(echo $PR_INFO | jq -r .author)" >> $GITHUB_ENV
+          echo "pr_mergedBy=$(echo $PR_INFO | jq -r .mergedBy)" >> $GITHUB_ENV
+          echo "PR Author: $(echo $PR_INFO | jq -r .author)"
+          echo "PR MergedBy: $(echo $PR_INFO | jq -r .mergedBy)"
```

1. Let's not assign this to environment variable if its just arbitrary information rather than something another tool consumes automatically.
2. Instead `echo`ing the value separately, let's use `tee`

```suggestion
          echo "author=$(echo ${PR_INFO} | jq -r .author)" | tee --append $GITHUB_OUTPUT
          echo "merger=$(echo ${PR_INFO} | jq -r .merger)" | tee --append $GITHUB_OUTPUT
```

With this, you can now access the information with `${{ steps.pr-info.outputs.author }}`.